<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayCommunication</class>
 <widget class="QWidget" name="DisplayCommunication">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>944</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Communication Traffic</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_start">
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_clear">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_save">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_copy">
        <property name="text">
         <string>Copy</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="box_stop_on_error">
        <property name="text">
         <string>Stop On Error</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="box_timestamp">
        <property name="text">
         <string>Timestamp</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="text_traffic"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
